packages/app/background-jobs-common/src/background-job-processor/managers/LokaliseQueueManager.ts

```diff
@@ -0,0 +1,62 @@
+import type { JobsOptions, Queue, QueueOptions } from 'bullmq'
+import type { BullmqQueueFactory } from '../factories/index.ts'
+import { commonBullDashboardGroupingBuilder } from '../public-utils/index.ts'
+import { QueueManager } from './QueueManager.ts'
+import type { QueueConfiguration, QueueManagerConfig, SupportedJobPayloads } from './types.ts'
+
+export type LokaliseQueueConfiguration<
+  ModuleId extends string = string,
+  QueueOptionsType extends QueueOptions = QueueOptions,
+  JobOptionsType extends JobsOptions = JobsOptions,
+> = Omit<QueueConfiguration<QueueOptionsType, JobOptionsType>, 'bullDashboardGrouping'> & {
+  moduleId: ModuleId
+}
+
+/**
+ * Lokalise-specific queue manager that automatically handles bull dashboard grouping.
+ *
+ * - Automatically builds dashboard grouping as [serviceId, moduleId]
+ * - Enables lazy initialization in production (disabled in tests)
+ */
+export class LokaliseQueueManager<
```

Comment:
I wouldn't use Lokalise in the name, it's a reasonable default for anyone without specific needs.

Maybe let's call it ModuleAwareQueueManager?